lib/Worker/DefaultPool.php

```diff
@@ -68,8 +68,6 @@ public function __construct(int $maxSize = self::DEFAULT_MAX_SIZE, WorkerFactory
         $busyQueue = $this->busyQueue;
 
         $this->push = static function (Worker $worker) use ($workers, $idleWorkers, $busyQueue) {
-            \assert($workers->contains($worker), "The provided worker was not part of this queue");
-
```

Comment:
Why did you remove this assertion?

Reply:
See issue #66 for more details.
1. Pool autoremove all stopped workers.
2. PooledWorker::_destroy return worker back to pool
3. Because of that assert will fail and phpunit tests also.

Reply:
Maybe we shouldn't auto-remove all stopped workers?